app/views/application/equality-monitoring/ethnic-background.html

```diff
@@ -97,9 +65,9 @@
       }, {
         text: "Caribbean"
       }, {
-        text: "Another Black background",
+        text: "Any other Black, African or Caribbean background",
```

Comment:
The Census used `Any other Black, Black British or Caribbean background` for this option

Reply:
I took the wording from the [Equality information pattern](https://design-system.service.gov.uk/patterns/equality-information/).

Reply:
Will stick with the wording from the GDS pattern for now, and raise this issue upstream.